interface/main/messages/save.php

```diff
@@ -0,0 +1,246 @@
+<?php
+/**
+ * /interface/main/messages/save.php
+ *
+ * @package MedEx
+ * @link    http://www.MedExBank.com
+ * @author  MedEx <support@MedExBank.com>
+ * @copyright Copyright (c) 2017 MedEx <support@MedExBank.com>
+ * @license https://www.gnu.org/licenses/agpl-3.0.en.html GNU Affero General Public License 3
+ */
+
+require_once "../../globals.php";
+require_once "$srcdir/acl.inc";
+require_once "$srcdir/lists.inc";
+require_once "$srcdir/forms.inc";
+require_once "$srcdir/patient.inc";
+require_once "$srcdir/MedEx/API.php";
+require_once("$srcdir/formatting.inc.php");
```

Comment:
TODO (again, this is for me)
remove above line since formatting.inc.php is always autoloaded in globals.php